converter/converters/koki_rc_to_kube_v1_rc.go

```diff
@@ -41,13 +41,6 @@ func Convert_Koki_ReplicationController_to_Kube_v1_ReplicationController(rc *typ
 		if len(kubeSpec.Template.Labels) == 0 {
```

Comment:
Update the comment above this if condition.  The code is no longer ensuring there is at least 1 label in the template and selector, but instead is ensuring that a label is set will match a selector if one is provided.

Reply:
Removed the comments